src/api/client.js

```diff
@@ -18,10 +25,31 @@ function assetTypesFromState(assetTypes) {
   return Object.keys(assetTypes).filter(key => assetTypes[key]);
 }
 
-export function requestAssets(courseId, { page = 0, pageSize = 50, sort = 'sort', assetTypes = {} }) {
-  const assetTypesToFilter = assetTypesFromState(assetTypes);
+function getDatabaseAttributesFromAssetAttributes(sort) {
+  return assetAttributesToDatabaseAttributes[sort];
+}
+
+function objectIsNonEmpty(object) {
+  return Object.keys(object).length > 0;
+}
+
+export function requestAssets(courseId, params) {
```

Comment:
The function now takes an object called params that contains the request parameters.